mp/src/game/server/filters.cpp

```diff
@@ -10,6 +10,8 @@
 #include "ai_squad.h"
 #include "ai_basenpc.h"
 
+#include "neo_player.h"
```

Comment:
We'd had the general convention of guarding NEO specific blocks of code inside Valve implementation with the:
```cpp
#if NEO
// Here's some NT code
#endif // NEO
```
similar to what Valve does with their games that do this.

So since we are not inside a purely NT unit here, this should be:
```cpp
#if NEO
#include "neo_player.h"
#endif
```
or similar.

Obviously we are only building for NT and not for other Source games, but abiding by this convention lets us locate NT-specific stuff in the code base more easily.